tests/data/sample.sdrf.tsv

```diff
@@ -0,0 +1,5 @@
+source name	characteristics[organism]	characteristics[organism part]	characteristics[disease]	characteristics[cell name]	characteristics[biological replicate]	assay name	technology type	comment[technical replicate]	comment[fraction identifier]	comment[label]	comment[data file]	comment[instrument]	comment[modification parameters]	comment[cleavage agent details]	comment[fragment mass tolerance]	comment[precursor mass tolerance]
```

Comment:
Why `characteristics[cell name]` instead of `characteristics[cell type]`?  we used `characteristics[cell type]` in the schema yaml.